test/commands/scanner/rule/add.test.ts

```diff
@@ -15,7 +15,7 @@ describe('scanner:rule:add', () => {
 
 	describe('E2E', () => {
 		const myLanguage = 'apex';
-		describe('Test Case: Adding individual JARs', () => {
+		it('Test Case: Adding individual JARs', () => {
```

Comment:
A bunch of the unit tests didn't have 'it' blocks. This causes the `before` methods to be skipped, the `before` methods are what sets up the ioc container.